bin/pt-online-schema-change

```diff
@@ -8412,7 +8412,7 @@ my $term_readkey = eval {
 
 use sigtrap 'handler', \&sig_int, 'normal-signals';
 
-
+my $plugin;
```

Comment:
Why did you make this variable global? It is enough to have it defined inside the main function.

Reply:
Gotcha: to use it in the sub _die.